esim/README.md

```diff
@@ -4,9 +4,9 @@ This [W3ID](https://w3id.org) provides a persistent URI namespace for the ESIM (
 
 ## Uses
 
-ESIM (Energy System Information Model) is a domain-specific metadata model that is represented as an RDF graph 
-and that describes information about energy systems at building and urban levels including their automation and
-control systems. It comprises functional, structural and physical descriptions of the
+ESIM (Energy System Information Model) is a domain-specific metadata model, is represented as an RDF graph,
```

Comment:
I really did mean for the "is" in "is represented" to be deleted... for the comma after "model" to replace "that is".

Reply:
You are right. I made a small PR #3548 to correct this.